x/incentive/keeper/payout.go

```diff
@@ -47,7 +47,56 @@ func (k Keeper) ClaimReward(ctx sdk.Context, addr sdk.AccAddress, multiplierName
 		return err
 	}
 
-	k.ZeroClaim(ctx, claim)
+	k.ZeroUSDXMintingClaim(ctx, claim)
+
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			types.EventTypeClaim,
+			sdk.NewAttribute(types.AttributeKeyClaimedBy, addr.String()),
+			sdk.NewAttribute(types.AttributeKeyClaimAmount, claim.Reward.String()),
```

Comment:
Add claim type to the event

Reply:
https://github.com/Kava-Labs/kava/pull/780/commits/ac6d43f4968d631eb510ab7cd7fee7ae36bdfc7f